### ISSUE TITLE
Enumerated type is not exported

### ISSUE THREAD

**Issue by gasrodriguez** [SystemRDL/PeakRDL-systemrdl#2]
Hi, I'm using the ip-exact importer and exporting to systemrdl, and the enumerated definition is not being exported.

- input file `my_subblock.xml`
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!--Generated by PeakRDL IP-XACT-->
<ipxact:component xmlns:ipxact="http://www.accellera.org/XMLSchema/IPXACT/1685-2014" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.accellera.org/XMLSchema/IPXACT/1685-2014 http://www.accellera.org/XMLSchema/IPXACT/1685-2014/index.xsd">
  <ipxact:vendor>example.org</ipxact:vendor>
  <ipxact:library>mylibrary</ipxact:library>
  <ipxact:name>my_subblock</ipxact:name>
  <ipxact:version>1.0</ipxact:version>
  <ipxact:memoryMaps>
    <ipxact:memoryMap>
      <ipxact:name>my_subblock_mmap</ipxact:name>
      <ipxact:addressBlock>
        <ipxact:name>my_subblock</ipxact:name>
        <ipxact:baseAddress>'h0</ipxact:baseAddress>
        <ipxact:range>'h110</ipxact:range>
        <ipxact:width>32</ipxact:width>
        <ipxact:register>
          <ipxact:name>reg1</ipxact:name>
          <ipxact:addressOffset>'h0</ipxact:addressOffset>
          <ipxact:size>32</ipxact:size>
          <ipxact:field>
            <ipxact:name>f1</ipxact:name>
            <ipxact:bitOffset>0</ipxact:bitOffset>
            <ipxact:enumeratedValues>
              <ipxact:enumeratedValue>
                <ipxact:name>EMPTY</ipxact:name>
                <ipxact:displayName>EMPTY</ipxact:displayName>
                <ipxact:description>RX-FIFO empty</ipxact:description>
                <ipxact:value>'h0</ipxact:value>
              </ipxact:enumeratedValue>
              <ipxact:enumeratedValue>
                <ipxact:name>NOT_EMPTY</ipxact:name>
                <ipxact:displayName>NOT_EMPTY</ipxact:displayName>
                <ipxact:description>RX-FIFO not empty.</ipxact:description>
                <ipxact:value>'h1</ipxact:value>
              </ipxact:enumeratedValue>
            </ipxact:enumeratedValues>
            <ipxact:resets>
              <ipxact:reset>
                <ipxact:value>'h0</ipxact:value>
              </ipxact:reset>
            </ipxact:resets>
            <ipxact:bitWidth>1</ipxact:bitWidth>
            <ipxact:volatile>true</ipxact:volatile>
            <ipxact:access>read-write</ipxact:access>
          </ipxact:field>
          <ipxact:field>
            <ipxact:name>f2</ipxact:name>
            <ipxact:bitOffset>8</ipxact:bitOffset>
            <ipxact:resets>
              <ipxact:reset>
                <ipxact:value>'h7b</ipxact:value>
              </ipxact:reset>
            </ipxact:resets>
            <ipxact:bitWidth>8</ipxact:bitWidth>
            <ipxact:volatile>true</ipxact:volatile>
            <ipxact:access>read-write</ipxact:access>
          </ipxact:field>
        </ipxact:register>
      </ipxact:addressBlock>
    </ipxact:memoryMap>
  </ipxact:memoryMaps>
</ipxact:component>
```
- command
```
peakrdl systemrdl my_subblock.xml -o my_subblock.rdl
```
- output file `my_subblock.rdl`
```rdl
addrmap my_subblock__my_subblock_mmap {
    addrmap {
        reg {
            regwidth = 0x20;
            field {
                sw = rw;
                encode = f1_enum_t;
            } f1[0:0] = 0x0;
            field {
                sw = rw;
            } f2[15:8] = 0x7B;
        } reg1 @ 0x0;
    } my_subblock @ 0x0;
};
```

Field `f1` has the `encode` property pointing to the expected enum type, but the type is not defined.
I'm mostly sure the problem is with the systemrdl exporter because exporting from ip-xact to ip-xact or from systemrdl (with the enum added by hand) to ip-xact show the values correctly.

**Comment by amykyta3** [comment]
Thanks!
Yep looks like I forgot to actually generate the enum definition. 
I put this exporter together pretty quickly so not surprised I missed a few things!

**Comment by amykyta3** [comment]
Fixed in v0.3.0
